### DISCUSSION TITLE
Allow primitive values in `where` clauses

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#6148] [Feature Requests & Ideas]
I've seen a few people do queries like this:

```
query Posts($title: String!) {
  Posts(where: { title: $title }) {
    totalDocs
  }
}
```

where they expect `title` to accept primitive values and getting errors (because it doesn't).

Is there a way in GraphQL for these sort of queries to accept either a where object (so like how we do it currently) or a primitive value (in which case it will default to an equals check)?

This also applies to the Local API (I don't use the REST API often I don't know if this is an issue over there)

**Comment by cbratschi** [comment]
Good idea, Mongoose for instance supports such shortcuts too:
https://masteringjs.io/tutorials/mongoose/find

I would love to see RegEx support for texts. We use this a lot in other projects.
